app/models/setting/general.rb

```diff
@@ -15,7 +15,7 @@ def self.default_settings
       self.set('max_trend', N_("Max days for Trends graphs"), 30, N_('Max trends')),
       self.set('db_pending_migration', N_("Should the `foreman-rake db:migrate` be executed on the next run of the installer modules?"), true, N_('DB pending migration')),
       self.set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
-      self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60, N_('Proxy request timeout')),
+      self.set('proxy_request_timeout', N_("Open, read and write timeout for HTTP requests from Foreman to Smart Proxy (in seconds)"), 60, N_('Smart Proxy request timeout')),
```

Comment:
Rest client doesn't actually have a write timeout- which makes sense as this is http requests and not IO (so even a post will timeout if the response isn't read in time) - https://github.com/rest-client/rest-client/blob/master/lib/restclient/request.rb#L36-L38

Reply:
Sure done.